CMakeLists.txt

```diff
@@ -105,6 +105,11 @@ else()
     set(TR_STABLE_RELEASE 1)
 endif()
 
+# derived from above: lib version
+math(EXPR TR_LIB_VERSION_MAJOR "${TR_VERSION_MAJOR} + 14714")
+math(EXPR TR_LIB_VERSION_MINOR "${TR_VERSION_MINOR}")
+math(EXPR TR_LIB_VERSION_PATCH "${TR_VERSION_PATCH} * 100 + 0${TR_STABLE_RELEASE} * 99 + 0${TR_VERSION_BETA_NUMBER}")
```

Comment:
- the *99 isn't intuitive on first read; examples in the comments would help
- the `PATCH` suffix is maybe misnamed, since it's more than just patch
- the `TR_LIB_` name is a little confusing, since it's a bundle number unrelated to libtransmission
- we _only_ use these variables in building the CFBundleVersion string; can we just build that here and unset these temporaries?

```suggestion
# derived from above: CFBundleVersion
# note: CFBundleVersion only honors 3 numbers,
# so third number has to hold both patch and beta info.
# 5.0.0         -> 14719.0.99
# 5.0.0-beta.1  -> 14719.0.101
# 5.0.1         -> 14719.0.199
math(EXPR CFBUNDLE_1 "${TR_VERSION_MAJOR} + 14714")
math(EXPR CFBUNDLE_2 "${TR_VERSION_MINOR}")
math(EXPR CFBUNDLE_3 "${TR_VERSION_PATCH} * 100 + 0${TR_STABLE_RELEASE} * 99 + 0${TR_VERSION_BETA_NUMBER}")
set(CFBUNDLE_VERSION "${CFBUNDLE_1}.${CFBUNDLE_2}.${CFBUNDLE_3}")
unset(CFBUNDLE_1)
unset(CFBUNDLE_2)
unset(CFBUNDLE_3)
```

Reply:
You mixed up the examples, but then, it means that good examples were needed. :)
fd8a11602ed6f883646dcbf4c47d7fd27dd19c52